src/it/unibs/ing/ingsw/config/ConfigView.java

```diff
@@ -0,0 +1,120 @@
+        configController = new ConfigController(config);
+    }
+
+    public void printConfig(){
+        System.out.println(configController.getConfigAsString());
+    }
+
+    public void createConfig() {
+        // FIXME: solo una volta la piazza, tipo if(!existsPiazza...)
+        String piazza = InputDati.leggiStringaNonVuota("Inserisci piazza di scambio definitiva: ");
+        List<String> luoghi = inserisciLuoghi();
+        List<DayOfWeek> days = inserisciGiorni();
+        List<TimeInterval> timeIntervals = inserisciIntervalliOrari();
+        int deadline = InputDati.leggiInteroConMinimo("Inserisci la deadline: ", 1);
+        configController.makeConfig(piazza, luoghi, days, timeIntervals, deadline);
+    }
+
+    public void execute() {
+        MyMenu mainMenu = new MyMenu("Gestione configurazione", new String[] {
+                "Visualizza configurazione",
+                "Crea configurazione"
+        });
+
+        int scelta;
+        do {
+            scelta = mainMenu.scegli();
+            switch (scelta) {
+                case 1 -> printConfig();
+                case 2 -> createConfig();
+            }
+        }while (scelta != 0);
+    }
+
+    /**
+     * Inserisce dall'UI una lista di luoghi
+     * @return Una lista di luoghi
+     */
+    private List<String> inserisciLuoghi() {
+        List<String> luoghi = new ArrayList<>();
+        boolean continua = true;
+        do {
+            String luogo = InputDati.leggiStringaNonVuota("Inserisci luogo di scambio: ");
+            if (luoghi.contains(luogo)) {
+                System.out.println("Luogo già presente :(");
+            } else {
+                luoghi.add(luogo);
+                continua = InputDati.yesOrNo("Vuoi inserire un altro luogo? ");
+            }
+        } while(continua || luoghi.isEmpty());
+
+        return luoghi;
+    }
+
+    /**
+     * Stampa i giorni della configurazione
+     */
+    private void printDays() {
+        configController.getDays().forEach(dayOfWeek -> {
+            System.out.printf("%s ", dayOfWeek.getDisplayName(TextStyle.FULL, Locale.ITALIAN));
+        });
+    }
+
+    /**
+     * Inserisce dall'UI una lista di giorni
+     * @return Una lista di giorni
+     */
+    private List<DayOfWeek> inserisciGiorni() {
+        printDays();
+
+        List<DayOfWeek> days = new ArrayList<>();
+        boolean continua = true;
+        do {
+            DayOfWeek day = DayOfWeek.of(InputDati.leggiIntero("Inserisci un giorno della settimana [1-7]: ", 1, 7));
+            if (days.contains(day)) {
+                System.out.println("Giorno già presente :(");
+            } else {
+                days.add(day);
+                continua = InputDati.yesOrNo("Vuoi inserire un altro giorno? ");
+            }
+        } while(continua || days.isEmpty());
+
+        return days;
+    }
+
+    private List<TimeInterval> inserisciIntervalliOrari() {
+        // TODO: Mancano controlli per intervalli sovrapposti nella lista
```

Comment:
Tutto qui, magari bisognerà mettere un nuovo metodo in `TimeInterval` per aiutarci (tipo `between` o `contains`)